src/server/scripts/Northrend/Ulduar/HallsOfStone/brann_bronzebeard.cpp

```diff
@@ -267,23 +278,48 @@ class brann_bronzebeard : public CreatureScript
 
             GameObject* go = nullptr;
             if (headMask & 0x1) // Kaddrak
-                if ((go = me->GetMap()->GetGameObject(pInstance->GetGuidData(GO_KADDRAK))))
-                    activate ? go->SendCustomAnim(0) : go->SetGoState(GO_STATE_READY);
-
+                if ((go = me->GetMap()->GetGameObject(pInstance->GetGuidData(GO_KADDRAK)))) {
```

Comment:
Braces go to the next line https://www.azerothcore.org/wiki/cpp-code-standards#brackets

![image](https://github.com/user-attachments/assets/5041d507-5765-4db7-9936-1d31d7880931)